src/Microsoft.VisualStudio.ProjectSystem.Managed/ProjectSystem/Properties/InterceptedProjectProperties/ApplicationPropertyPage/CompoundTargetFrameworkValueProvider.cs

```diff
@@ -50,6 +50,11 @@ private async Task<ComplexTargetFramework> GetStoredPropertiesAsync()
             return await GetStoredComplexTargetFrameworkAsync(configuration);
         }
 
+        public override Task<IEnumerable<string>> GetProjectFileMSBuildPropertyNamesAsync(string propertyName)
```

Comment:
>IEnumerable [](http://example.com/codeflow?start=29&length=11)

Why IEnumerable<string> instead of just string?

Reply:
there can be multiple defined properties that one value provider sets @jacdavis